### PR TITLE
Input and image

### PR DESCRIPTION
**Pull request by romabulani**
Added only input text fields, let me know if other fields are required to be added

### CODE REVIEW

File: Components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.img-responsive{
+    height: auto;
+    width: 60%;
+}
+
+.img-round{
+    border-radius: 50%;
+}
+
+@media only screen and (max-width : 768px){
```
Comment (shrey27) [inline Components/image/image.css:19]: Try to follow mobile first approach. Go with min-width:768px

**Review by shrey27 — APPROVED** [review]

File: Components/image/image.css (hunk 1 above)
Comment (romabulani) [reply Components/image/image.css:19]: As far as I remember, it was mentioned that both approaches are fine. If 80% users are using the desktop version, we should go for desktop first and similarly for mobile first approach.

File: Components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@import url('../../globalVar.css');
+.input-container{
+    align-items: flex-start;
+    column-gap: 1rem;
+    display: flex;
```
Comment (anandsr21harshit) [inline Components/input/input.css:5]: Always write display first. How will align-items and other flex property will work if you write them before display:flex? Not good for debugging purpose.

File: Components/image/image.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+@import url('../../globalVar.css');
+.img-container{
+    align-items: center;
+    column-gap: 1rem;
+    display: flex;
```
Comment (anandsr21harshit) [inline Components/image/image.css:5]: Same thing. Write display: flex first before using other flex-properties.

File: Components/input/input.css (hunk 2 above)
Comment (romabulani) [reply Components/input/input.css:5]: Hi Harshit,
I am following alphabetical wise ordering of css properties for all the css files, as it was suggested in one of the CSS lectures

**Review by anandsr21harshit — APPROVED** [review]
All seems good from my end.

**Pull request merged**